2010-10-04  Michael Tautschnig  <mt@debian.org>

	* examples/simple/disk_config/FAIBASE: Added bootable:1 as requested by
		Maximiliano Curia (apparently still necessary on some systems).
	* examples/simple/package_config/GNOME: Remove gnome-apt, ice*-gnome-support
		as these don't exist in squeeze anymore (thanks Karl Wallner for reporting
		this)
Index: trunk/examples/simple/disk_config/FAIBASE
===================================================================
--- trunk.orig/examples/simple/disk_config/FAIBASE
+++ trunk/examples/simple/disk_config/FAIBASE	
@@ -2,7 +2,7 @@
 #
 # <type> <mountpoint> <size>   <fs type> <mount options> <misc options>
 
-disk_config disk1 disklabel:msdos fstabkey:uuid
+disk_config disk1 disklabel:msdos bootable:1 fstabkey:uuid
 
 primary /      250      ext3  rw,noatime,errors=remount-ro
 logical swap   200-1000 swap  rw                   
Index: trunk/examples/simple/package_config/GNOME
===================================================================
--- trunk.orig/examples/simple/package_config/GNOME
+++ trunk/examples/simple/package_config/GNOME	
@@ -2,10 +2,7 @@
 
 iceweasel
 icedove
-iceweasel-gnome-support
-icedove-gnome-support
 menu gdm
-gnome-apt
 gnome-core
 desktop-base
 evince gconf-editor
